sycl/plugins/unified_runtime/pi2ur.hpp

```diff
@@ -4496,8 +4496,8 @@ piextCommandBufferCreate(pi_context Context, pi_device Device,
   ur_device_handle_t UrDevice = reinterpret_cast<ur_device_handle_t>(Device);
   ur_exp_command_buffer_desc_t UrDesc;
   UrDesc.stype = UR_STRUCTURE_TYPE_EXP_COMMAND_BUFFER_DESC;
-  UrDesc.isInOrder = ur_bool_t(false);
-  UrDesc.enableProfiling = ur_bool_t(true);
+  UrDesc.isInOrder = Desc->is_in_order;
+  UrDesc.enableProfiling = Desc->enable_profiling;

```

Comment:
`pi_bool` is [32-bits](https://github.com/intel/llvm/blob/sycl/sycl/include/sycl/detail/pi.h#L215), `ur_bool_t` is [8-bits](https://github.com/oneapi-src/unified-runtime/blob/main/include/ur_api.h#L352), so I think this change would be prudent:

```suggestion
  UrDesc.isInOrder = ur_bool_t(Desc->is_in_order);
  UrDesc.enableProfiling = ur_bool_t(Desc->enable_profiling);
```